provider/cmd/pulumi-resource-gcp/bridge-metadata.json

```diff
@@ -26114,6 +26115,7 @@
             "gcp:organizations/getFolder:getFolder": 0,
             "gcp:organizations/getFolders:getFolders": 0,
             "gcp:organizations/getIAMPolicy:getIAMPolicy": 0,
+            "gcp:organizations/getIamPolicy:getIamPolicy": 0,
```

Comment:
I feel worried about these two side-by-side entries that differ only in casing.

```
          "gcp:organizations/getIAMPolicy:getIAMPolicy": 0,
            "gcp:organizations/getIamPolicy:getIamPolicy": 0,
```

Are these really distinct tokens?

Reply:
I've changed the purpose of this PR to resolve this issue. I'll open a new PR to remove the patches.